tools/build-on-openbsd

```diff
@@ -16,12 +22,45 @@ pkgs="
    py3-setuptools
    py3-yaml
    sudo--
+   wget
 "
-[ -f "$depschecked" ] || pkg_add "${pkgs}" || fail "install packages"
 
-touch $depschecked
+[ -f $depschecked ] || echo "Installing the following packages: $pkgs"; output=$(pkg_add -zI $pkgs 2>&1)
+
+
+if  echo "$output" | grep -q -e "Can't find" -e "Ambiguous"; then
+    echo "Failed to find or install one or more packages"
+    echo "Failed Package(s):"
+    echo "$output"
+    exit 1
+else
+    echo Successfully installed packages
+    touch $depschecked
+
+    python3 setup.py build
+    python3 setup.py install -O1 --distro openbsd --skip-build --init-system sysvinit_openbsd
 
-python3 setup.py build
-python3 setup.py install -O1 --distro openbsd --skip-build
+    echo "Installation completed."
+    RC_LOCAL="/etc/rc.local"
+    RC_LOCAL_CONTENT="
+/usr/lib/cloud-init/ds-identify
+wait
 
-echo "Installation completed."
+cloud-init init --local
+wait
+
+cloud-init init
+wait
+
+cloud-init modules --mode config
+wait
+
+cloud-init modules --mode final
+"
+    if ! test -e $RC_LOCAL; then
+        echo "export PATH=$PATH:/usr/local/sbin:/usr/local/bin" >> $RC_LOCAL
+        echo "$RC_LOCAL_CONTENT" >> $RC_LOCAL
+    elif ! grep -Fq "cloud-init" $RC_LOCAL; then
```

Comment:
```suggestion
    if ! grep -Fq "PATH" $RC_LOCAL; then;
        echo "export PATH=$PATH:/usr/local/sbin:/usr/local/bin" >> $RC_LOCAL
    fi
    if ! grep -Fq "cloud-init" $RC_LOCAL; then
```

Reply:
@igalic @holmanb Drafting this again...uggh.  I thought I had this updated and working and pushed it up, but it's not. @igalic  I had thought I was making it better by just removing `PATH` altogether, and it worked for the most part, but looking at the logs further it complained it couldn't find `wget` again.

Reply:
@igalic OK, I have reverted to back to what it was.  Unless there is a strong reason to update with your suggestion, I think I'd just like to keep it the way it was.  I tested that and it seemed to work, and I need to work on some other things for a bit.